src/Views/Form/Templates/Sequoia/views/receipt.php

```diff
@@ -122,8 +123,10 @@
     </div>
 
 <?php
+
+$formId     = FrontendFormTemplateUtils::getFormId();
```

Comment:
@glaubersilva Set form id for `the_title` filter will be wrong because In GiveWP receipt view (`Donation Confirmation page`) is a page.

The same statement is true for the default receipt view and classic template receipt view.